helm-chart/todoapp/templates/rbac.yml

```diff
@@ -0,0 +1,30 @@
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: {{ .Values.todoapp.RBAC.name }}
+  namespace: mateapp
```

Comment:
The namespace is hardcoded as `mateapp`. Consider using a templated value like `{{ .Values.todoapp.namespace }}` to ensure consistency and flexibility across different environments.